packages/lite-xl/build.sh

```diff
@@ -0,0 +1,17 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/lite-xl/lite-xl
+TERMUX_PKG_DESCRIPTION="A lightweight text editor written in Lua"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@leapofazzam123"
+TERMUX_PKG_VERSION=2.0.3
+TERMUX_PKG_SRCURL="https://github.com/lite-xl/lite-xl/archive/refs/tags/v$TERMUX_PKG_VERSION.tar.gz"
+TERMUX_PKG_SHA256=6c8a4ea284f102f772e3aff583236e89d5b1171664526dd501000b681ae5c4e2
+TERMUX_PKG_DEPENDS="sdl2, freetype, liblua52"
+
+termux_step_pre_configure() {
+	# fix meson not detecting librt
+	echo 'INPUT(-lc)' > $TERMUX_PREFIX/lib/librt.so
+}
+
+termux_step_post_make_install() {
+	rm $TERMUX_PREFIX/lib/librt.so
+}
```

Comment:
Patch build system instead of creating & deleting files that are not necessary.

Reply:
Ok but the library detection happens in the subproject and we can't easily patch them because meson downloads it and it's not a git submodule

Reply:
So I need to create the file and deleting it after build